tests/functionnal/vagrant/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -xe
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# common steps
+$SCRIPT_DIR/common.sh
+
+cp $SCRIPT_DIR/vbox.yaml reservation.yaml
+
+virtualenv venv
+. venv/bin/activate
+pip install -r requirements.txt
+
+python -m enos.enos up &&\
+python -m enos.enos os &&\
+python -m enos.enos init &&\
```

Comment:
It's safer to go with `enos deploy` rather than `enos up && enos os && enos init` because the `enos init` will eventually get a new name.